tensorflow/compiler/xla/service/gpu/BUILD

```diff
@@ -972,13 +973,14 @@ cc_library(
         "//tensorflow/compiler/xla:xla_data_proto_cc",
         "//tensorflow/compiler/xla/hlo/ir:hlo",
         "//tensorflow/tsl/platform:statusor",
+        "//tensorflow/tsl/platform:tensor_float_32_hdr_lib",
```

Comment:
This dependency would move to the if_cuda_is_configured section.

Reply:
LGTM.